thingsvision/core/extraction/extractors.py

```diff
@@ -39,18 +39,20 @@
 
 class TorchvisionExtractor(PyTorchExtractor):
     def __init__(
-            self,
-            model_name: str,
-            pretrained: bool,
-            device: str,
-            model_path: str = None,
-            model_parameters: Dict[str, Union[str, bool, List[str]]] = field(
-                default_factory=lambda: {}
-            ),
-            preprocess: Optional[Callable] = None,
+        self,
+        model_name: str,
+        pretrained: bool,
+        device: str,
+        model_path: str = None,
+        model_parameters: Dict[str, Union[str, bool, List[str]]] = field(
+            default_factory=lambda: {}
+        ),
+        preprocess: Optional[Callable] = None,
     ) -> None:
         model_parameters = (
-            model_parameters if model_parameters else {"weights": "DEFAULT"}
```

Comment:
It may be worth considering using these default values (for weights and alignment) not only when no model parameters are given, but also when they do not contain those keys.